src/pages/Thanks/ThanksPage.vue

```diff
@@ -352,6 +358,17 @@ export default {
 				);
 			}
 		}
+
+		if (this.$appConfig.enableOptimizely) {
```

Comment:
Generally we're tracking these type of transaction events here: https://github.com/kiva/ui/blob/main/src/plugins/kv-analytics-plugin.js#L214

Thus, you should be able to skip any changes to the Thanks page, import your new event util (or just embed that one method) and use the transactionData we've already aggregated for these events.

Reply:
+1